mod/wikirate_source/set/type/source/source_preview.rb

```diff
@@ -1,19 +1,18 @@
 format :html do
-
-  view :preview ,:tags=>:unknown_ok do |args|
-    url_card = card.fetch(:trait=>:wikirate_link)
+  view :preview ,tags: :unknown_ok do |args|
+    url_card = card.fetch(trait: :wikirate_link)
     url = url_card ? url_card.item_names.first : nil
-    args.merge!({:url=>url})
+    args[:url] = url
     wrap args do
-    [
-      render_navigation_bar(args),#render in structure source_preview_nav_bar_structure
-      render_hidden_information(args),
-      render_source_preview_container(args)
-    ]
+      [
+        render_navigation_bar(args),# render in structure source_preview_nav_bar_structure
```

Comment:
Line is too long. [90/80]<br>Space missing after comma.<br>Put a space before an end-of-line comment.